r2/r2/lib/pages/pages.py

```diff
@@ -4029,6 +4031,11 @@ def __init__(self, user, site, link=None):
         else:
             choices = []
 
+        if admin:
+            admintempl = FlairTemplate()
+            admintempl.text = "ModFlair"
+            choices.append(target_wrapper(admintempl))
```

Comment:
This is essentially, a "FakeFlairTemplate"; and there is no worry about it's id being none; as this only occurs for mods / admins; and those inputs never require an ID.